requirements/pip_tools.txt

```diff
@@ -20,7 +20,7 @@ tomli==2.0.1
     # via
     #   build
     #   pep517
-wheel==0.37.1
+wheel==0.38.2
```

Comment:
Dropped support for `Python 3.6`: https://wheel.readthedocs.io/en/stable/news.html